gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
```

Comment:
This should be a separate PR, and should just upgrade all the other dependencies too

Reply:
Given that we're often updating the gradle wrapper for e.g. Geyser in PRs that don't solely update dependencies, I see no reason to not do it here too